rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -61,3 +62,135 @@ def test_makerdao_simple_transaction(
             extra_data={'vault_type': 'ETH-A'},
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29']])  # noqa: E501
+def test_makerdao_sdai_redeem_transaction(
+        database,
+        ethereum_inquirer,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c
+    """
+    tx_hash = deserialize_evm_tx_hash('0x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c')  # noqa: E501
+
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    assert events == [
+        EvmEvent(
+            event_identifier='10x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c',
+            sequence_index=0,
+            timestamp=1695854591000,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal('0.001301015216220134'), usd_value=ZERO),
+            location_label='0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29',
+            notes='Burned 0.001301015216220134 ETH for gas',
+            tx_hash=tx_hash,
+            counterparty='gas',
+            address=None,
+            extra_data=None,
+        ), EvmEvent(
+            event_identifier='10x6b2a1f836cfc7c28002e4ac60297daa6d79fcde892d9c3b9ca723dea2f21af5c',
+            sequence_index=176,
+            timestamp=1695854591000,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=A_DAI,
+            balance=Balance(amount=FVal('16.601085935411927527'), usd_value=ZERO),
```

Comment:
I don't like repeating myself so please do the stuff I said, and are repeating all over the test

```suggestion
            balance=Balance(amount=FVal('16.601085935411927527')),
```